backend/src/appointment/migrations/versions/2024_06_25_2133-156b3b0d77b9_add_ftue_level_to_subscribers.py

```diff
@@ -0,0 +1,24 @@
+"""add ftue_level to subscribers
+
+Revision ID: 156b3b0d77b9
+Revises: 12c7e1b34dd6
+Create Date: 2024-06-25 21:33:27.094632
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '156b3b0d77b9'
+down_revision = '12c7e1b34dd6'
```

Comment:
Unfortunately this collides with previous developments 😬 
We had two migrations on 2024-06-13 which change the current migration head. I'll fix the head locally on my system, but we should provide a continuous migration queue.

Reply:
Yeah I haven't had the chance to rebase this off of main yet. I'll create a merge migration.

I'm not a fan of git-based database migrations for this reason :sweat_smile:

Reply:
Yeah, this is a pain during development... Thanks for a merge migration 🙏🏻